core/config/backend_config_loader.go

```diff
@@ -182,26 +185,16 @@ func (cl *BackendConfigLoader) GetAllBackendConfigs() []BackendConfig {
 	return res
 }
 
-func (cl *BackendConfigLoader) RemoveBackendConfig(m string) {
-	cl.Lock()
-	defer cl.Unlock()
-	delete(cl.configs, m)
-}
-
-func (cl *BackendConfigLoader) ListBackendConfigs() []string {

```

Comment:
Unlike most of the other moves, this function _was removed_. All places that used it have been converted to `GetAllBackendConfigs()` - the one place in live code essentially did that anyway, and the test has been modified.